src/Makevars.win

```diff
@@ -1,6 +1,19 @@
-# Makevars.win can be tested by uploading package to win-builder.r-project.org
+VERSION=4.4.1.1
+PKG_CPPFLAGS = -I../windows/netcdf-${VERSION}/include \
+	-DHAVE_LIBUDUNITS2 -DHAVE_DECL_NC_RENAME_GRP=1
 
-PKG_CPPFLAGS = -DHAVE_LIBUDUNITS2 -DHAVE_DECL_NC_RENAME_GRP=1 \
-  -Drestrict=restrict
```

Comment:
I was using `restrict` deliberately, and it worked in my previous testing. Is there a reason to remove it?